src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -370,69 +401,81 @@ export default class PhotoField extends React.Component {
 
   setCropBox = () => {
     window.requestAnimationFrame(() => {
-      // center the cropbox using container width
-      const containerWidth = this.refs.cropper.getContainerData().width;
-      // const containerWidth = this.refs.container.offsetWidth;
-      const smallScreen = isSmallScreen(this.state.windowWidth) || onReviewPage(this.props.formContext.pageTitle);
-      const left = smallScreen ? (containerWidth / 2) - 120 : (containerWidth / 2) - 150;
-      this.refs.cropper.setCropBoxData({
-        top: 0,
-        left,
-        height: smallScreen ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE,
-        width: smallScreen ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE
-      });
-
-      // use the cropbox dimensions to force canvas into default position
-      const { height: cropBoxHeight, width: cropBoxWidth, left: cropBoxLeft } = this.refs.cropper.getCropBoxData();
-      const { width: oldCanvasWidth, height: oldCanvasHeight, naturalHeight, naturalWidth } = this.refs.cropper.getCanvasData();
-      // tall images take the full width of the cropBox
-      if (naturalHeight > naturalWidth) {
-        this.refs.cropper.setCanvasData({
-          width: cropBoxWidth,
-          left: cropBoxLeft,
-          top: 0
-        });
-        // wide images take the full height of the cropBox
-        // to center, uses the ratio of cropBoxHeight / cavasHeight
-      } else {
-        this.refs.cropper.setCanvasData({
-          height: cropBoxHeight,
+      // sometimes this callback is called before the new cropper has mounted
+      // check if the cropper instance has canvasData before setting its values
+      if (!_.isEmpty(this.refs.cropper.getCanvasData())) {
```

Comment:
I noticed this was causing an error in the console- nothing visual. Sometimes this callback is executed while the component is unmounting and these values aren't ready. Since `setCropBox` is called again once the component is initialized, can safely by pass this when the data isn't available.